### PR TITLE
Remove esnI18nInterpolator

### LINKED ISSUE #10: Remove esnI18nInterpolator
Due to [this issue](https://github.com/OpenPaaS-Suite/esn-frontend-calendar/issues/1), we need to remove the old `esnI18nInterpolator` in order to make the new one, angularjs translate style works.

### PR DESCRIPTION
**Pull request by ppppbn**

### CODE REVIEW

**Review by RockyMeadow — DISMISSED** [review]

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+dist/
+venv/
+tmp/
+*.log
+.idea
+.vscode/
+.floo
+.flooignore
+package-lock.json
+.DS_Store
```
Comment (tuanlc) [inline .gitignore]: Check this comment: https://github.com/OpenPaaS-Suite/esn-frontend-calendar/pull/5/files#r450718701

**Review by tuanlc — DISMISSED** [review]

File: .gitignore (hunk 1 above)
Comment (ppppbn) [reply .gitignore]: Thanks, I updated it

**Review by tuanlc — COMMENTED** [review]
Missing the issue number in the second commit

**Review by piavgh — APPROVED** [review]

**Comment by ppppbn** [comment]
> Missing the issue number in the second commit

**Review by tuanlc — APPROVED** [review]

**Pull request merged**